reana/reana_dev/git.py

```diff
@@ -363,12 +379,29 @@ def git_fork(component, exclude_components, browser):  # noqa: D301
                 browser, "".join([" -c {0}".format(c) for c in component])
             )
         )
+
     for component in components:
-        cmd = "{0} https://github.com/reanahub/{1}/fork;".format(browser, component)
-        click.echo(cmd)
-    click.echo(
-        'echo "Please continue the fork process in the opened' ' browser windows."'
-    )
+        if automatic:
+            cmd = f"gh repo fork reanahub/{component} --clone=false"
+            run_command(cmd)
+            display_message(
+                f"Repository {component} forked successfully using GitHub CLI.",
+                component,
+            )
+        else:
+            display_message(
```

Comment:
`display_message` cannot be used when `--automatic` is not provided, as the line prefix (i.e. date and component) printed by `display_message` interferes with eval when running `eval "$(reana-dev git-fork -c ALL -b firefox)"`

Reply:
Right - my bad! Reverted it to `click.echo`.